insonmnia/miner/overseer.go

```diff
@@ -276,20 +277,19 @@ func (o *overseer) handleStreamingEvents(ctx context.Context, sinceUnix int64, f
 					s <- pb.TaskStatusReply_BROKEN
 					close(s)
 				}
-				if c.description.CommitOnStop {
-					go func() {
+				go func() {
```

Comment:
Hey, it's the separate change!

Reply:
Nope, because this prevents cleanup to be called on container resulting in hanging tinc containers and networks